versions.lock

```diff
@@ -37,34 +37,36 @@ com.sun.jersey:jersey-json:1.9 (5 constraints: 945f2f90)
 com.sun.jersey:jersey-server:1.9 (4 constraints: ef373c01)
 com.sun.jersey.contribs:jersey-guice:1.9 (4 constraints: 65529ed7)
 com.sun.xml.bind:jaxb-impl:2.2.3-1 (1 constraints: 330c2404)
+com.tdunning:json:1.8 (1 constraints: 910b72db)
```

Comment:
Another JSON library! Can you check whether we need it?

Reply:
Introduced by hive-common. Seems not necessary, I will remove this. 

```
com.tdunning:json:1.8
\--- org.apache.hive:hive-common:2.3.6
     \--- org.apache.hive:hive-serde:2.3.6
          \--- org.apache.hive:hive-metastore:2.3.6
```